### ISSUE TITLE
Windows 10 BSOD When Using DrRacket

### ISSUE THREAD

**Issue by AssossaGPB** [racket/drracket#121]
I installed Racket v6.9 (x86_64) yesterday on my Windows 10 desktop. I can use it just fine, everything functions. But after opening it, Windows 10 will consistently BSOD after a random amount of time. I've had it BSOD after having DrRacket open only a few seconds and today I had it open a good 20 minutes or so before it BSOD'd. My computer does not BSOD with any other programs or tasks, it seems to only be DrRacket.

### Here are my specs if it helps:
i7 2600k @ 4.3GHz (Overclocked, but stable with everything else)
16GB DDR3-1600 RAM (4 Modules)
2x 1TB HDD (7200rpm & 5200rpm)
Zotac AMP! GTX 1060 6GB (Not overclocked)

Windows 10 Home
Version 1703
OS Build 15063.296

### [BSOD Dump File](https://drive.google.com/file/d/0B7-Z_fZnBrvGMm50czB0VGRfOG8/view?usp=sharing)

**Comment by mfelleisen** [comment]
Are you aware of `https://github.com/racket/racket/issues/1671` 

(This is a Window bug but the discussion tells you more.)

**Comment by AssossaGPB** [comment]
No, I didn't see that, thanks. I'm installing the snapshot and will report back if I get a crash with it.

Should I close this issue since it's already been reported?

**Comment by mfelleisen** [comment]
Please report back and close the issue then.
